src/renderer/wg_engine/tvgWgBindGroups.cpp

```diff
@@ -334,49 +409,40 @@ void WgBindGroupBlit::initialize(WGPUDevice device, WGPUQueue queue, WGPUSampler
 }
 
 
-void WgBindGroupBlit::release()
+void WgBindGroupTextureSampled::release()
 {
     releaseBindGroup(mBindGroup);
 }
 
 //************************************************************************
-// bind groups pools
+// bind group pools
 //************************************************************************
 
-WgBindGroupOpacity* WgBindGroupOpacityPool::allocate(WgContext& context, uint32_t opacity)
-{
-    WgBindGroupOpacity* bindGroup{};
-    if (mPool.count == 0) {
-        bindGroup = new WgBindGroupOpacity;
-        bindGroup->initialize(context.device, context.queue, opacity);
-        mUsed.push(bindGroup);
-        mList.push(bindGroup);
-    } else {
-        bindGroup = mPool.last();
-        bindGroup->update(context.device, context.queue, opacity);
-        mUsed.push(bindGroup);
-        mPool.pop();
-    }
-    return bindGroup;
+void WgBindGroupOpacityPool::initialize(WgContext& context)
+{
+    memset(mPool, 0x00, sizeof(mPool));
```

Comment:
hmm this looks wrong... you might wanna this:
```
mPool.reserve(256);
memset(mPool.data, 0x00, sizeof(WgBindGroupOpacity*) * 256);
```

Reply:
No, now we are using simple array, without any wrapper.
 `WgBindGroupOpacity* mPool[256];`